gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://apicast.io/poolicy-v1/schema#manifest#",
+  "name": "oauth2 token introspection policy",
+  "description": 
+  ["This policy executes OAuth 2.0 Token Introspection",
+   "(https://tools.ietf.org/html/rfc7662) for every API calls."],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "introspectionUrl": {
+        "description": "Introspection Endpoint URL",
+        "type": "string"
+      },
+      "clientId": {
```

Comment:
Looks like this name is not the same as in the tests/code. Everywhere it uses `client_id`.